### PR TITLE
feat: add footer

### LINKED ISSUE #32: Add Footer section

### PR DESCRIPTION
**Pull request by aanabdulrohman**
aduh

### CODE REVIEW

File: src/components/Footer.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+export default function Footer() {
+  return (
+    <footer className="bg-slate-950 text-white py-8 px-4 border-t-4 border-blue-600 w-full mt-9">
+      <div className="max-w-md px-4 ">
```
Comment (nnivxix) [inline src/components/Footer.tsx]: use grid instead of flex

File: src/components/Footer.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+        <h1 className="text-2xl font-bold mb-4">Vilm</h1>
+        <p className="text-gray-400 mb-1 ">
+          The Vilm website is a movie discovery platform featuring a modern
+          design. Users can browse various movie titles and access detailed
+          information, including the synopsis and trailers.
+        </p>
+        <div className="flex items-center mb-4">
+          <h3 className="text-gray-400">Data provided by </h3>
+          <img src="tmdb.png" alt="" width={"80px"} className="ml-2" />
```
Comment (nnivxix) [inline src/components/Footer.tsx]: add link to provider API

https://www.themoviedb.org/

use the component `<Link>`

File: src/components/Footer.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+        <div className="col-spans-1">
+          <h1 className="text-2xl font-bold mb-4">Vilm</h1>
+          <p className="text-gray-400 mb-1 ">
+            The Vilm website is a movie discovery platform featuring a modern
+            design. Users can browse various movie titles and access detailed
+            information, including the synopsis and trailers.
+          </p>
+          <div className="flex items-center mb-4">
+            <h3 className="text-gray-400">Data provided by </h3>
+            <a href="https://www.themoviedb.org/">
```
Comment (nnivxix) [inline src/components/Footer.tsx]: use tag `<Link></Link>`

**Comment by aanabdulrohman** [comment]
mark as resolved

**Review by nnivxix — APPROVED** [review]

**Pull request merged**